src/components/cart.lineitem.jsx

```diff
@@ -141,6 +144,25 @@ class CartLineItem extends React.Component {
     );
   }
 
+  renderConfiguration() {
+    const { item } = this.props;
+    const keys = Object.keys(item.configuration);
+    if (keys) {
+      return keys.map(key => (
+        <li className="configuration" key={key.split(configurationPrefix)[1]}>
```

Comment:
Trying this out. Why is it necessary to even provide a configurationPrefix? It's sufficient enough to allow this to be set through CM and display the same value in the storefront as below.
```suggestion
        <li className="configuration" key={key}>
        <label htmlFor="option-name" className="option-name">
            {key}
        </label>
```
![image](https://user-images.githubusercontent.com/39800563/47866116-4843ac80-ddd4-11e8-9b0b-be2a9506a81c.png)

Reply:
You are right, when Mark created the modifier input, he added `SIG_` as a prefix, so I figured out it was like a convention within EP. I should've asked, I fixed it!